HW35_SlotMachineGame.py

```diff
@@ -0,0 +1,56 @@
+"""
+  Slot machine game
+"""
+class SlotMachineGame:
+
+    def __init__(self, balance: int, isMember = None, freeGame = True):
```

Comment:
For Python using snake_case (isMember -> is_member) and do't add whitespaces around default argument values